jatszohaz/jatszohaz/views.py

```diff
@@ -65,6 +67,8 @@ class ProfileView(LoginRequiredMixin, DetailView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        context['comment_form'] = NewCommentForm()
+        context['comments'] = Comment.objects.all().order_by('created').filter(user=self.get_object())

```

Comment:
Illetve még egy dolog, a kommentek védelmére be lehetne tenni egy jogosultság ellenőrzést is, hogy ha mezei bejelentkezett felhasználóval van dolgunk (és nem körtaggal), akkor ne is kérjük le ezeket az adatokat.
`self.request.user.has_perm('jatszohaz.view_all')` elvileg működni fog.